tests/e2e/utils/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Added
 
 - Factories for variable product, variation, and grouped product
+- New function to create orders by batch using the orders API
```

Comment:
All utilities functions are listed in the [package readme](https://github.com/woocommerce/woocommerce/tree/trunk/tests/e2e/utils#readme). Whenever we add a new utility the PR should add the function to the list.

Reply:
Done. Fixed in https://github.com/woocommerce/woocommerce/pull/30184/commits/16083c257478115f613f2e28366a8d16d2f3525f